github/ci/prow-deploy/files/orgs.yaml

```diff
@@ -542,6 +546,14 @@ orgs:
           - brybacki
         repos:
           kubevirt-velero-plugin: admin
+      containerized-data-importer-api-maintainers:
+        description: ""
+        maintainers:
+          - mhenriks
+          - awels
+          - aglitke
+        repos:
+          containerized-data-importer-api: admin
```

Comment:
Is this a temporary measure or will this stay? In general we want to configure GitHub repositories as much as possible using peribolos. So maybe write access would be enough here?